keanu-python/keanu/vertex/label.py

```diff
@@ -9,6 +10,24 @@
 
 class _VertexLabel(JavaObjectWrapper):
```

Comment:
We might want to write a comment saying this is for internal use, so people don't try to pass it to `Vertex`.

Reply:
I've added a docstring:

        class _VertexLabel(JavaObjectWrapper):
        """
        This class is intended for private use - as a user you should only need to specify string values.
        It's used behind the scenes e.g. in :class:`keanu.vertex.lambda_model.LambdaModel`
        """